modules/prebidServerBidAdapter.js

```diff
@@ -666,6 +683,10 @@ export function PrebidServer() {
       .reduce(utils.flatten)
       .filter(utils.uniques);
 
+    if (_s2sConfig && _s2sConfig.syncEndpoint) {
+      queueSync(_s2sConfig.bidders, bidRequests[0].gdprConsent);
```

Comment:
Is `bidRequests` guaranteed to have at least one element in it? If not, this could break.

Reply:
Per this check, it should always at least have 1 element:
https://github.com/prebid/Prebid.js/blob/master/src/adaptermanager.js#L289

But I can add another check in this code to be sure.

Reply:
Either way.

My only two concerns are "does it work now?" and "is it likely that the code changes to break it in the future without anyone realizing?" Looks like the answer to (1) is "yes." (2) usually depends on how thorough the docs and unit tests are.

Your call in the end.